api/nodes/serializers.py

```diff
@@ -272,23 +273,31 @@ class NodeLinksSerializer(JSONAPISerializer):
 
     id = IDField(source='_id', read_only=True)
     type = TypeField()
-    target_node_id = ser.CharField(source='node._id', help_text='The ID of the node that this Node Link points to')
+    target_node_id = ser.CharField(write_only=True, source='node._id', help_text='The ID of the node that this Node Link points to')
```

Comment:
Target node info doesn't need to be in two places.  Here, I need target_node_id to create the relationship, but I am returning the target_node information under relationships. In the other PR mentioned in this ticket, target_node_id won't be under attributes in the request either,  but will go under relationships.  It will be under relationships in the request, and under relationships in the response, as soon as both PR's are merged.